explore/src/main/scala/explore/proposal/ProposalTabContents.scala

```diff
@@ -91,14 +90,14 @@ object ProposalTabContents:
 
   private val component = ScalaFnComponent[Props]: props =>
     for {
-      ctx      <- useContext(AppContext.ctx)
-      readonly <-
-        useMemo((props.programDetails.get.proposalStatus, props.userIsReadonlyCoi)):
-          (status, roCoi) =>
-            status === ProposalStatus.Submitted || status === ProposalStatus.Accepted || roCoi
+      ctx <- useContext(AppContext.ctx)
```

Comment:
I guess you could do just `useContext(AppContext.ctx).map: ctx =>`

Reply:
True. I'm not sure that is clearer, though.